sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -70,20 +80,26 @@ export class SupportingUserController {
       >
     >,
   ): Promise<MustReturnJobActionAcknowledgement> {
-    const supportingUser =
-      await this.supportingUserService.getSupportingUserById(
-        job.variables.supportingUserId,
-      );
-    if (!supportingUser) {
-      job.error(
-        SUPPORTING_USER_NOT_FOUND,
-        "Supporting user not found while checking for supporting user response.",
+    const jobLogger = new Logger(job.type);
+    try {
+      const supportingUser =
+        await this.supportingUserService.getSupportingUserById(
+          job.variables.supportingUserId,
+        );
+      if (!supportingUser) {
+        const message =
+          "Supporting user not found while checking for supporting user response.";
+        jobLogger.error(message);
+        job.error(SUPPORTING_USER_NOT_FOUND, message);
+      }
+      const outputVariables = filterObjectProperties(
+        supportingUser.supportingData,
+        job.customHeaders,
       );
+      jobLogger.log("Supporting user data loaded.");
```

Comment:
Why didn't we use logger service like other places? 

![image](https://github.com/bcgov/SIMS/assets/54600590/4d54dc85-a793-43f3-b6b5-cf40d3cf2b5f)

Reply:
This would create a logger at the "class" level, which would be okay also, but I believe that the jobs can produce their own logs.
For instance, in the way that it is right now, it puts the `job.name` in the format below (in yellow).
![image](https://github.com/bcgov/SIMS/assets/61259237/2b5cc60c-562b-48ae-8564-184cd1f99a5b)